network_analysis/closest-facility-static/src/main/java/com/esri/samples/closest_facility_static/ClosestFacilityStaticSample.java

```diff
@@ -86,10 +91,10 @@ public void start(Stage stage) throws Exception {
       ProgressIndicator progressIndicator = new ProgressIndicator();
       progressIndicator.setVisible(true);
 
-      // create a ArcGISMap with a Basemap instance with an Imagery base layer
-      ArcGISMap map = new ArcGISMap(Basemap.createStreetsWithReliefVector());
+      // create a map with a basemap style
```

Comment:
In some samples we specify which basemap style, do we want to do this as a standard?

Reply:
Mostly the logic was that we'd include it if it said it previously and not if it didn't. In this case I just removed it as it was incorrect anyway. If we want it to be consistent one way or the other I can go back through. One for @Rachael-E to add to and see what we think.